LocalSettings.php

```diff
@@ -2305,9 +2305,11 @@
 	],
 
 	// Restriction types
-	'+wgRestrictionLevels' => [
+	'wgRestrictionLevels' => [
```

Comment:
Needs to stay +wgRestrictionLevels otherwise all wikis who have been specified here will find breakages.

(+ means that it'll use the default + what ever the wiki specifies).

Reply:
As far as I know, using `+` in front of configuration variable means merging settings with `DefaultSettings.php`, but in order to put `user` before `autoconfirmed` and `sysop`, the core default value must be overridden since the core default value of it is `["","autoconfirmed","sysop"]`.
Merging of wiki-specific additional level and default settings is done by using `+` in front of **wiki database name** (not variable), so for example value of `$wgRestrictionLevels` for [큰숲백과](https://bigforest.miraheze.org/) (`bigforestwiki`) will be `["","user","autoconfirmed","sysop","editvoter"]`.
Please review this again, and if it still needs `+`, I will push a new commit to prepend it.

Reply:
Paladox is correct you must add + afaik

Reply:
Well, won't the `default` value of `$wgRestrictionLevels` be weird like `['','autoconfirmed','sysop','','user','autoconfirmed','sysop']` if I add `+`?